packages/Path-View-Core.package/SClassDiagram.class/instance/extendDiagramAreaTo..st

```diff
@@ -0,0 +1,10 @@
+geometry
+extendDiagramAreaTo: newBounds
+
+	| bounds |
+	bounds := self boundsInWorld.
+	(bounds top > newBounds top) ifTrue: [self dragTopBy: (bounds top - newBounds top)].
```

Comment:
coordinate system origin is top left. so y1 < y2 essentially means y1 is top of y2, which then means the window was extended